frontend/src/component/location/input.vue

```diff
@@ -140,11 +137,11 @@ export default {
     },
   },
   watch: {
-    lat() {
-      this.updateCoordinateInput();
-    },
-    lng() {
-      this.updateCoordinateInput();
+    latlng: {
+      handler() {
+        this.updateCoordinateInput();
+      },
+      deep: true,
```

Comment:
Have you checked whether the deep flag is strictly needed here? I was hoping that the Array Proxy would automatically trigger the event when one of the values changed, but of course that may not be the case...